sklearn/callback/_callback_context.py

```diff
@@ -123,7 +126,7 @@ class CallbackContext:
     """
 
     @classmethod
-    def _from_estimator(cls, estimator, *, task_name, task_id, max_subtasks=None):
```

Comment:
Why did you change this signature ?
(some of my previous comments are irrelevant with this change)